tests/test_venue_request.py

```diff
@@ -180,8 +180,8 @@ def test_venue_deployment(self, client, selenium, request_page, helpers):
             subject='A request for service has been submitted'
         )
         assert messages and len(messages) == 1
-        assert messages[0]['content']['text'].startswith('A request for service has been submitted. Check it here')
-
+        #assert messages[0]['content']['text'].startswith('A request for service has been submitted. Check it here') 
+        assert messages[0]['content']['text'].startswith('A request for service has been submitted by TestVenue@OR2021. Check it here') # Changed by me
```

Comment:
you don't need to comment the line here, we have the git history to know that it was you :)